.github/workflows/survey-on-merged-pr.yml

```
@@ -46,4 +48,4 @@ jobs:
           USERNAME=$(jq -r '.pull_request.user.login' "$GITHUB_EVENT_PATH")
           gh pr comment ${PR_NUM} --repo open-telemetry/opentelemetry-java-instrumentation --body "Thank you for your contribution @${USERNAME}! 🎉 We would like to hear from you about your experience contributing to OpenTelemetry by taking a few minutes to fill out this [survey](${SURVEY_URL})."
         env:
-          GH_TOKEN: ${{ steps.otelbot-token.outputs.token }}
\ No newline at end of file
+          GH_TOKEN: ${{ steps.otelbot-token.outputs.token }}
```

```suggestion
          GH_TOKEN: ${{ steps.otelbot-token.outputs.token }}
          
```

a lot of lints required the new line at the end

---

this PR adds a newline at the end (there wasn't one previously)